crates/parcel_plugin_transformer_js/src/transformer/conversion.rs

```diff
@@ -507,6 +514,18 @@ fn convert_dependency(
     )),
     _ => {
       let mut env = asset.env.clone();
+      base_dependency.set_kind(format!("{}", transformer_dependency.kind));
```

Comment:
This one is a bit dodgy. Happy to write a match if we'd prefer, however it'd just map to the exact enum values.